Dockerfile-linux.template

```diff
@@ -201,6 +201,12 @@ RUN set -eux; \
 {{ ) else "" end -}}
 	go version
 
+{{ if [ "1.20" ] | index(env.version | rtrimstr("-rc")) then "" else ( -}}
+# don't auto-upgrade the gotoolchain
+# https://github.com/docker-library/golang/issues/472
+ENV GOTOOLCHAIN=local
```

Comment:
We might want to move this up a bit higher (later) to make sure our `go` invocations during install and verification don't somehow accidentally download. 👀